### PR TITLE
Viviali iotcentral new API onboarding

### PR DESCRIPTION
**Pull request by viv-liu**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by yaohaizh** [comment]
@ruowan do you know what kind of this error https://dev.azure.com/azure-sdk/public/_build/results?buildId=164674&view=logs&j=f10f5372-40ed-54fc-caa4-b1cb5bd1252b&t=d3b2b841-a93b-5fa5-47c3-d8c4f6eae6e3&l=258 and how to fix it?

File: specification/iotcentral/data-plane/readme.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+This block is updated by an automatic script. Edits may be lost!
+
+``` yaml $(tag) == 'all-api-versions' /* autogenerated */
+# include the azure profile definitions from the standard location
+require: $(this-folder)/../../../profiles/readme.md
+
+# all the input files across all versions
+input-file:
+  - $(this-folder)/preview/2019-10-28/iotcentral.json
```
Comment (ruowan) [inline specification/iotcentral/data-plane/readme.md]: This path is not correct, pls change it to 
`$(this-folder)/Microsoft.IoTCentral/preview/2019-10-28/iotcentral.json`

**Comment by ruowan** [comment]
> @ruowan do you know what kind of this error https://dev.azure.com/azure-sdk/public/_build/results?buildId=164674&view=logs&j=f10f5372-40ed-54fc-caa4-b1cb5bd1252b&t=d3b2b841-a93b-5fa5-47c3-d8c4f6eae6e3&l=258 and how to fix it?

I add a comment. The input file path is not correct, so avocado can't find the input file and report these errors.

File: specification/iotcentral/data-plane/readme.md (hunk 1 above)
Comment (rafarhat) [reply specification/iotcentral/data-plane/readme.md]: thanks   @ruowan  this is passing now.  The only remaining issue is the breaking change validation it seems oad is failing to de-serialize the swagger to do the check , but when i run the tool locally it passes fine.

**Comment by ruowan** [comment]
@qiaozha Could you please take a look at the breaking change error?

**Comment by viv-liu** [comment]
We have approval from API review board @rysweet for public preview

**Review by yaohaizh — APPROVED** [review]

**Comment by yaohaizh** [comment]
@viv-liu merged

**Pull request merged**